scripts/create_test_subset.py

```diff
@@ -11,7 +11,7 @@
 This example will populate acute-care-test with the metamist data for 4 families.
 """
 
-from typing import Dict, List, Optional, Tuple
+from typing import Dict, List, Optional, Tuple, Set
```

Comment:
You can use inbuilt types now, eg:

```
dict[str, str]
set[int]
list[float]
str | None
```

Not super big deal, but good to refactor these out as we go